src/libcore/cell.rs

```diff
@@ -207,8 +207,8 @@ use ptr;
 ///
 /// # Examples
 ///
-/// Here you can see how using `Cell<T>` allows to use mutable field inside
-/// immutable struct (which is also called 'interior mutability').
+/// In this example, you can see that `Cell<T>` enables mutation inside an
+/// immutable struct. In other words, it enables interior mutability.
```

Comment:
This seems like an improvement, except that you should still keep the explicit quoting on `'interior mutability'` to make it clear that it defines this new term.